tests/sentry/api/endpoints/test_project_commits.py

```diff
@@ -0,0 +1,64 @@
+from django.urls import reverse
+
+from sentry.models import Commit, Release, ReleaseCommit, ReleaseProject, Repository
+from sentry.testutils import APITestCase
+
+
+class ProjectCommitListTest(APITestCase):
```

Comment:
Small testing nits:
You can add `endpoint = "sentry-api-0-project-commits"` as a class variable here.

Then when fetching responses, you can use
```
response = self.get_success_response(self.project.organization.slug, self.project.slug)
assert [r["id"] for r in response.data]  == [commit.key]
```

Reply:
When I try doing it that way, this test fails because `response.data` is an empty list. I added `endpoint` as a class variable, removed `url` and replaced `response = self.client...` with `response = self.get_success_response...`

Am I missing something? 

```
class ProjectCommitListTest(APITestCase):
    endpoint = "sentry-api-0-project-commits"

    def test_simple(self):
        project = self.create_project(name="komal")
        version = "1.1"
        repo = Repository.objects.create(organization_id=project.organization_id, name=project.name)
        release = Release.objects.create(organization_id=project.organization_id, version=version)
        commit = self.create_commit(repo=repo, project=project, key="a" * 40, release=release)
        ReleaseProject.objects.create(project=project, release=release)

        self.login_as(user=self.user)

        response = self.get_success_response(self.project.organization.slug, self.project.slug)
        assert [r["id"] for r in response.data] == [commit.key]
```

Reply:
Sorry, missed that you created your own project at the top - should be `self.get_success_response(`project.organization.slug`, `project.slug`)`

Reply:
Thanks Dan!